core/src/main/scala/org/apache/spark/sql/delta/DeltaConfig.scala

```diff
@@ -191,9 +192,24 @@ trait DeltaConfigsBase extends DeltaLogging {
 
     val globalConfs = entries.asScala.flatMap { case (key, config) =>
       val sqlConfKey = sqlConfPrefix + config.key.stripPrefix("delta.")
-      Option(sqlConfs.getConfString(sqlConfKey, null)) match {
-        case Some(default) => Some(config(default))
-        case _ => None
+      val sqlConf = Option(sqlConfs.getConfString(sqlConfKey, null))
+      val deprecatedSqlConfKey = deprecatedSqlConfPrefix + config.key.stripPrefix("delta.")
+      val deprecatedSqlConf = Option(sqlConfs.getConfString(deprecatedSqlConfKey, null))
+      (sqlConf, deprecatedSqlConf) match {
+        case (Some(default), Some(deprecatedDefault)) =>
+          (default, deprecatedDefault) match {
+            case (`deprecatedDefault`, _) => Some(config(deprecatedDefault))
+            case _ =>
+              logConsole(
+                s"You are setting property $sqlConfKey to $default and " +
+                 "deprecated property $deprecatedSqlConfKey to $deprecatedDefault " +
```

Comment:
The variables $deprecatedSqlConfKey, $deprecatedDefault, and $deprecatedSqlConfKey are not going to be shown if you don't add the prefix "s" on each line that is being concatenated. I would recommend instead using the three quotes string(s""" $x """) to only add the prefix once.

Reply:
Got, I will change it.